lib/relay.js

```diff
@@ -204,7 +204,10 @@ function responseHandler(filterRules, config) {
         const status = (response && response.statusCode) || 500;
         logContext.httpStatus = status;
         logContext.httpHeaders = response.headers;
-        logger.info(logContext, 'sending response back to websocket connection');
+
+        const logFunc = (status >= 400) ? logger.info : logger.debug;
```

Comment:
Please reverse this to:
`const logFunc = (status <= 200) ? logger.debug : logger.info;`

This will support the unlikely case where `status` is `NaN`, and actually log this case at `info` level.
This is because JS is ✨ , and `NaN <= 200` is `false`.

Reply:
Also, I think you need the same treatment for the `sending response back to HTTP connection` which is elsewhere.